client/state/posts/test/actions.js

```diff
@@ -252,7 +252,7 @@ describe( 'actions', () => {
 
 	describe( '#editPost()', () => {
 		it( 'should return an action object for a new post', () => {
-			const action = editPost( {
+			const action = editPost( 2916284, undefined, {
```

Comment:
I think this should work as-is, but one of my only worries with making this change is that `getEditorPostId` will return `null` for a new post, so we'll either need to handle both cases (which I believe we do) or try to normalize via e.g. setting a fallback `null` value for the action creators:

```js
export function savePost( siteId, postId = null, post ) {
```

What do you think?